Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/QuadBendingFEMForceField.inl

```diff
@@ -145,9 +145,9 @@ void QuadBendingFEMForceField<DataTypes>::initSmall(int i, Index&a, Index&b, Ind
   const  VecCoord& initialPoints = (this->mstate->read(core::ConstVecCoordId::restPosition())->getValue());
   qinfo->IntlengthElement = (initialPoints)[b] - (initialPoints)[a];
   qinfo->IntheightElement = (initialPoints)[d] - (initialPoints)[a];
-  qinfo->Intcentroid = ((initialPoints)[a] + (initialPoints)[c])/2.0f;
-  qinfo->Inthalflength = (sqrt(IntlengthElement[0]*IntlengthElement[0]+IntlengthElement[1]*IntlengthElement[1]+IntlengthElement[2]*IntlengthElement[2]))/2.0f;
-  qinfo->Inthalfheight = (sqrt(IntheightElement[0]*IntheightElement[0]+IntheightElement[1]*IntheightElement[1]+IntheightElement[2]*IntheightElement[2]))/2.0f;
+  qinfo->Intcentroid = ((initialPoints)[a] + (initialPoints)[c])/2;
```

Comment:
Is it a mistake to remove the .0 ?
By the way I see in all the rest of the pr that you removed all of the 'f' suffixes which make every floating value a double. Wouldn't it be more efficient and clean to create a suffix converting the value in Real ?

Reply:
It is not really possible to have a user-defined literal to convert to `Real`, because `Real` is a template parameter. However, we have the `_sreal` literal, which converts to `SReal`, but `Real` is not always equal to `SReal`. So `static_cast` is really the more rigorous solution.

I did some tests on implicit conversions of integer constants: https://godbolt.org/z/qcr5qGjnn
Based on the generated assembly code from the 3 compilers, I can conclude that `static_cast` and integer-defined constant are equivalent. I'll update the code because I find it more readable to use a simple integer compared to a `static_cast`.